docs/user_guide/simple_example.rst

```diff
@@ -18,20 +18,20 @@ This example shows ....
   # create btrack objects (with properties) from the segmentation data
   # (you can also calculate properties, based on scikit-image regionprops)
   objects = btrack.utils.segmentation_to_objects(
-              segmentation, properties=('area', )
-            )
+    segmentation, properties=('area', )
+  )
 
   # initialise a tracker session using a context manager
   with btrack.BayesianTracker() as tracker:
 
     # configure the tracker using a config file
-    tracker.configure_from_file('/path/to/your/models/cell_config.json')
+    tracker.configure('/path/to/your/models/cell_config.json')
 
     # append the objects to be tracked
     tracker.append(objects)
 
     # set the volume (Z axis volume is set very large for 2D data)
-    tracker.volume=((0, 1200), (0, 1600), (-1e5, 1e5))
+    tracker.volume = ((0, 1200), (0, 1600))
```

Comment:
Does this mean that `z`-axis values default to something?

Reply:
Yes, the z-axis defaults to those parameters now. I'll update the comment